test/unit/base.test.js

```diff
@@ -302,26 +303,27 @@ describe('CONVERT FUNCTION TESTS ', function() {
               .equal('{\n    "a": "example-a",\n    "b": "example-b"\n}');
             // Request header
             expect(rootRequest.header[0].value).to.equal('<integer>');
-            expect(exampleRequest.header[0].value).to.equal('header example');
+            expect(exampleRequest.header[0].value).to.equal(123);
             // Request query parameters
             expect(rootRequest.url.query[0].value).to.equal('<long> <long>');
```

Comment:
@VShingala why are we expecting '<long> <long>` here?

Reply:
@abhijitkane this was done to fix existing invalid examples in test schemas. If that's what you mean.